### ISSUE TITLE
Keyboard navigation does not work in grid with remote virtualization.

### ISSUE THREAD

**Issue by MayaKirova** [IgniteUI/igniteui-angular#1154]
### Description  
Keyboard navigation does not work in the grid with remote virtualization.

### Steps to reproduce  

1. Open the grid remote virtualization sample in demos dev.
2. Select a cell and start navigating using arrow down/arrow up.

### Result  
You cannot navigate outside of the currently loaded chunk using the keyboard.

### Expected result  
You should be able tonavigate outside of the currently loaded chunk using the keyboard.

**Comment by MayaKirova** [comment]
Already merged to master, probably with this merge PR: https://github.com/IgniteUI/igniteui-angular/pull/1200/commits
